src/dumbo/src/pdu/tcp.rs

```diff
@@ -512,7 +514,7 @@ impl<'a, T: NetworkBytesMut> TcpSegment<'a, T> {
             segment.bytes.htons_unchecked(OPTIONS_OFFSET + 2, value);
         }
 
-        segment_len += if let Some((payload_buf, max_payload_bytes)) = payload {
+        let payload_bytes_count = if let Some((payload_buf, max_payload_bytes)) = payload {
```

Comment:
Just wondering if there's any need for this change relative to the previous `segment_len += ...` assignment.

Reply:
I do not have a strong preference for this change, but while reading the code, I thought this would make it more clear.